opt/scopeserver/dataserver/modules/gserver/GServer.py

```diff
@@ -412,10 +423,16 @@ def getNextCluster(self, request, context):
             )
 
         f = self.get_features(loom=loom, query=cluster_metadata["description"])
+
+        for n, featureType in enumerate(f["featureType"]):
+            if featureType == f"Clustering: {clustering_meta['name']}":
+                clustering_index = n
+                break
```

Comment:
You could consider replacing this with a function, something like, `get_feature_index()`

Reply:
I'm going to leave this as is at the moment, because I would like to overhaul this in the future. Relying on the 'name' field is bad because there is no guarantee for uniqueness. We strip a lot of information when sending responses to the fronted, like the ID of the clustering, which would be very useful in these circumstances if we kept it. This would eliminate the need for this code or the suggested function. Will add an issue about this.